systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -893,6 +901,170 @@ void testConfigureDeploymentStrategy() {
+
+
+        internalClientSourceJob = internalClientSourceJob.toBuilder().withMessage("Producer B").build();
+        internalClientSourceJob.createAndWaitForReadiness(internalClientSourceJob.producerStrimzi().build());
+        ClientUtils.waitForClientSuccess(sourceProducerName, NAMESPACE, MESSAGE_COUNT);
+        LOGGER.info("Wait {} secs -> sync.group.offsets.interval.seconds, check msgs in target cluster", SYNC_GROUP_OFFSETS_INTERVAL_SECONDS);
+        waitForMM2OffsetIntervalMirroring(SYNC_GROUP_OFFSETS_INTERVAL_SECONDS);
+
+        internalClientTargetJob.createAndWaitForReadiness(internalClientTargetJob.consumerStrimzi().build());
+        ClientUtils.waitForClientSuccess(targetConsumerName, NAMESPACE, MESSAGE_COUNT);
+        JobUtils.deleteJobWithWait(NAMESPACE, sourceProducerName);
+        JobUtils.deleteJobWithWait(NAMESPACE, targetConsumerName);
+
+
+        LOGGER.info("Send 50 msgs to Source cluster");
+        internalClientSourceJob = internalClientSourceJob.toBuilder().withMessageCount(50).withMessage("Producer C").build();
+        internalClientSourceJob.createAndWaitForReadiness(internalClientSourceJob.producerStrimzi().build());
+        ClientUtils.waitForClientSuccess(sourceProducerName, NAMESPACE, 50);
+        JobUtils.deleteJobWithWait(NAMESPACE, sourceProducerName);
+
+        waitForMM2OffsetIntervalMirroring(SYNC_GROUP_OFFSETS_INTERVAL_SECONDS);
+
+        LOGGER.info("Receive 10 msgs from source cluster");
+        internalClientSourceJob = internalClientSourceJob.toBuilder().withMessageCount(10).build();
+        internalClientSourceJob.createAndWaitForReadiness(internalClientSourceJob.consumerStrimzi().build());
+        ClientUtils.waitForClientSuccess(sourceConsumerName, NAMESPACE, 10);
+        JobUtils.deleteJobWithWait(NAMESPACE, sourceConsumerName);
+
+        // wait N secs -> sync.group.offsets.interval.seconds, check msgs in target cluster
+        waitForMM2OffsetIntervalMirroring(SYNC_GROUP_OFFSETS_INTERVAL_SECONDS);
+
+        LOGGER.info("Receive 40 msgs from target cluster");
+        internalClientTargetJob = internalClientTargetJob.toBuilder().withMessageCount(40).build();
+//        internalClientTargetJob = internalClientTargetJob.toBuilder().withMessageCount(40).withAdditionalConfig(createNewConsumerCliendId("lala")).build();
+        internalClientTargetJob.createAndWaitForReadiness(internalClientTargetJob.consumerStrimzi().build());
+        ClientUtils.waitForClientSuccess(targetConsumerName, NAMESPACE, 40);
+        JobUtils.deleteJobWithWait(NAMESPACE, targetConsumerName);
+
+//        assertThat("Expected to receive 40 messages.", receivedMessagesTargetPart2 == 40);
+//        internalKafkaClientTarget.checkProducedAndConsumedMessages(sentMessagesSource, receivedMessagesTarget + receivedMessagesTargetPart2);
+    }
+
+    private String createNewConsumerCliendId(String prefix) {
+        return  "client.id=" + prefix + "-id-" + consumerCounter++;
+    }
```

Comment:
This is not needed FMPOV - you use it only in commented code, also you can use `ClientUtils.generateRandomConsumerGroup()` and pass it to `withConsumerGroup` or something.

Reply:
Yup, works without this anyway.